testing/ibus/ibus.install

```diff
@@ -0,0 +1,16 @@
+post_install() {
+	gtk-query-immodules-2.0 --update-cache
+	gtk-query-immodules-3.0 --update-cache
+
+	dconf update
+}
+
+post_upgrade() {
+	post_install
+}
+
+post_remove() {
+	post_install
+
+	rm -f etc/dconf/db/ibus
+} 
```

Comment:
no, this will not work. we use dedicated scripts instead of a catchall.

I also think we should implemenr the --update-cache thing as a trigger if possible.